lib/datasource/mock/mock_repository.dart

```diff
@@ -0,0 +1,30 @@
+import 'package:flutter/foundation.dart';
+import 'package:seeds/datasource/remote/model/account_guardians_model.dart';
+import 'package:seeds/datasource/remote/model/user_recover_model.dart';
+import 'package:seeds/domain-shared/result_to_state_mapper.dart';
+import 'package:async/async.dart';
+
+class MockRepository {
```

Comment:
are these mocks going to be used for testing purposes? Should they live in a testing package away from production code?